ex20.py

```diff
@@ -0,0 +1,33 @@
+from sys import argv
+
+script, input_file = argv
+
+def print_all(f):
+	print f.read()
+
```

Comment:
I type this code on debian's gedit, in gedit nothing automatic indentation, so the result like this...